spec/opensearch_helper.rb

```diff
@@ -16,15 +16,19 @@ def self.get_client(options)
     host_opts = { host: host, port: port, scheme: 'http' }
     ssl_opts = {}
 
+    if options[:user] && options[:password]
+      host_opts[:user] = options[:user]
+      host_opts[:password] = options[:password]
+      host_opts[:scheme] = 'https'
+      ssl_opts = { verify: false }
```

Comment:
do we need this line? can you check whether it works without this settings?

Reply:
yes, we need to add both. Skipping any of those settings is causing exceptions

```
host_opts[:scheme] = 'https' -> Caused by: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 
ssl_opts = { verify: false } -> Caused by: javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
```

https://github.com/naveentatikonda/logstash-input-opensearch/actions